app/views/curation_concerns/file_sets/_show_actions.html.erb

```diff
@@ -1,5 +1,5 @@
   <div class="form-actions">
-    <%= link_to "Download this File", main_app.download_path(presenter), class: 'btn btn-default' %>
+    <%= link_to "Download this File", main_app.download_path(presenter), class: 'btn btn-default', target: :_blank %>
```

Comment:
Why does this need to have a _blank target? Can't it open in the same window? I'm unclear as to how this is related to the other commit in the PR.